trieste/models/gpflow/sampler.py

```diff
@@ -44,6 +44,30 @@
 )
 
 
+def idealised_normal_samples(
+    batch_shape: tf.TensorShape, n_sample_dim: int, skip: int = 0
+) -> tf.Tensor:
+    """
+    Generates a batch of shape `batch_shape` sobol samples, skipping the first `skip`, where each
+    sample has dimension `n_sample_dim`.
+    """
+    sobol_samples = tf.math.sobol_sample(
+        dim=n_sample_dim,
+        num_results=tf.reduce_prod(batch_shape),
+        dtype=tf.float64,
+        skip=skip,
+    )
+
+    dist = tfp.distributions.Normal(
+        loc=tf.constant(0.0, dtype=tf.float64),
+        scale=tf.constant(1.0, dtype=tf.float64),
+    )
+    samples_shape = batch_shape + (n_sample_dim,)
+    normal_samples = tf.reshape(dist.quantile(sobol_samples), samples_shape)
```

Comment:
if we needed uniform random numbers then Sobol does the trick, but does this with quantile transform really works? the goal is to get normally distributed numbers that are more regular, analgous to the usual usage of Sobol sequence, right?

Reply:
would this work for `n_sample_dim>1`?

Reply:
@vpicheny @ChrisMorter you able to comment on this?

Reply:
gave it some further thought and tested it, it does work all right, for `n_sample_dim>1` as well

Reply:
Thanks. I might try to update the test to test with sample_dim = 2 then.